pandas/tests/extension/category/test_categorical.py

```diff
@@ -157,3 +160,13 @@ def test_value_counts(self, all_data, dropna):
 
 class TestCasting(base.BaseCastingTests):
     pass
+
+
+def test_combine():
```

Comment:
Should this be moved to be within the `TestMethods` class?  And can you reference the GitHub issue number in a comment?

Reply:
Fixed